service/controller/app/v1/resource/chartoperator/resource.go

```diff
@@ -213,7 +192,7 @@ func (r Resource) installChartOperator(ctx context.Context, cr v1alpha1.App) err
 		// Wait for chart-operator to be deployed. If it takes longer than
 		// the timeout the chartconfig CRs will be created during the next
 		// reconciliation loop.
-		b := backoff.NewConstant(30*time.Second, 5*time.Second)
+		b := backoff.NewConstant(20*time.Second, 10*time.Second)
```

Comment:
I will close my PR then.

Reply:
Thanks and yes I just copied these settings from your PR.